docs/04-guides/playback-a-video-stream.mdx

```diff
@@ -6,16 +6,166 @@ description: Playback video and livestreams
 import CodeBlock from "@theme/CodeBlock";
 import CodeBlockTabs from "@site/src/components/CodeBlockTabs";
 import CodeBlockTabItem from "@site/src/components/CodeBlockTabItem";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import Player from "@site/src/components/Player/Livepeer";
+import HTMLPlayer from "@site/src/components/Player/HTML";
 
 # Playback a video
 
-To playback your live or on demand video inside your application you'll need to
-use a video player component that supports the HLS protocol. While there are
-many HLS video players out there, we recommend using
-[livepeer.js](https://livepeerjs.org).
+This guide provides you the instructions to playback a video or a live stream on
+your website/applications. To playback live streams or on-demand video inside
+your application, you'll need to use a video player component that supports the
+HLS protocol.
 
-## Audio-only streams
+You can either use Livepeer's built-in player or your own player. This guide
+covers both options.
 
-If you want to provide only the audio version of the stream, you can use
-?video=false at the end of an .m3u8 URL to get an audio-only stream. For
-example, `https://livepeercdn.com/hls/{playbackId}/index.m3u8?video=none`
+## Using Livepeer's player
+
+Livepeer.js is a JavaScript library that allows you to upload, retrieve, and
+play assets. It provides a core vanilla JS library to easily connect to a
+Livepeer provider (e.g. Livepeer Studio), as well as React-specific
+hooks/components to provide memoization and DOM management.
+
+:::note
+
+Livepeer.js currently only supports applications built with React.js and
+Next.js.
+
+:::
```

Comment:
We should probably link to livepeer.js home page here as something like "for more information visit livepeerjs.org" or something like that. Could be just making the `Livepeer.js` hyperlinks to that page as well.

Or perhaps maybe in the end of this livepeer.js player section we should link to this guide: https://livepeerjs.org/react/Player

Then one could get more info on everything they could configure on the player etc. Could be right after the simple player example.

Reply:
FTR here was the old embeddable player docs
https://github.com/livepeer/livepeer-com/blob/36495e35a4ffa945cd206754139118136ca31fa5/packages/www/docs/guides/start-live-streaming/playback.mdx#L31